src/desktop/apps/article/components/App.js

```diff
@@ -51,7 +53,9 @@ export default hot(module)(
       return (
         <Fragment>
           <EditPortal article={article} />
-          {this.getArticleLayout()}
+          <ContextProvider currentUser={sd.CURRENT_USER}>
```

Comment:
Is this the most top-level entrypoint for `Article`?

Reply:
Yes, this is rendered from here: https://github.com/artsy/force/blob/master/src/desktop/apps/article/routes.js#L140